src/v3/src/transformer/field/transform.ts

```diff
@@ -53,15 +53,17 @@ export const transformStepInputs = (
     }, [])
     .filter((input) => input.visible !== false && input.mutable !== false)
     .reduce((acc: FormBag, input: Input) => {
-      const { name, required, mutable } = input;
+      const {
+        name, required, mutable, type,
+      } = input;
 
       // add uischema
       const uischema = mapUiElement(input);
       acc.uischema.elements = [...acc.uischema.elements, uischema];
 
       // add client validation for "required" field
       // do not validate immutable fields, they will always be added to payload programatically
-      if (required && mutable !== false) {
+      if (required && mutable !== false && type !== 'object') {
```

Comment:
fields have been flattened when get to this step, this check is to avoid credential object for stepper layout, like security question, and it would be separated to its own transformation step in later refactor PR